src/transformState.js

```diff
@@ -5,7 +5,22 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
```suggestion
  for (const action of actions) {
    switch (action.type) {
```
You can make destructuring to get all the needed props of action 
https://javascript.info/destructuring-assignment#object-destructuring